ucp/_libs/ucx_api.pyx

```diff
@@ -456,6 +539,15 @@ cdef class UCXEndpoint(UCXObject):
         assert self.initialized
         return int(<uintptr_t>self._handle)
 
+    def flush(self, cb_func, cb_args, cb_kwargs=dict()):
```

Comment:
Would suggest similar changes here as above. Namely using `None` and making `cb_args` also optional.